apps/src/StudioApp.js

```diff
@@ -2855,6 +2865,19 @@ StudioApp.prototype.polishGeneratedCodeString = function (code) {
 };
 
 /**
+ * Returns whether this view should be responsive based on level options.
+ * Responsive means that the visualizationColumn should resize as the
+ * window width changes, and that a grippy is available to manually resize
+ * the visualizationColumn.
+ */
+StudioApp.prototype.isResponsiveFromConfig = function (config) {
+  // use config.responsiveEmbedded because this.responsiveEmbedded may not be initialized yet
+  const isResponsiveEmbedView = !!(config.embed && config.responsiveEmbedded);
+  const isCodeView = !config.embed && !config.share;
```

Comment:
do we need to better differentiate this idea of code view from, for example, applab where we have code/design/data views? Maybe `isWorkspaceView`?